kpovel/tc-chat-back-prototype
# Pull Request: Simplify Email Verification Process

## Pull Request: Simplify Email Verification Process

### Summary:
This pull request simplifies the email verification process. Previously, after verifying their email address, users were required to make an additional API call to `/api/login` to receive access and refresh tokens. This PR changes that flow. Now, upon successful email verification through `/api/verification-user-email/{code}`, users will directly receive these tokens.

### Changes:
- Altered API documentation to reflect the changes in the email verification process.

### Why this is important:
This change streamlines the user registration and login process, making it less cumbersome for users to get started with our application. It also reduces the number of API calls needed for a common user flow, thereby improving the overall user experience.

### How to Test:
1. Register a new user account.
2. Trigger the email verification process.
3. Click on the verification link or copy the code and send a request to `/api/verification-user-email/{code}`.
4. Verify that the response includes access and refresh tokens.

### Additional Context:
- This change might affect other parts of the system that interact with the user authentication flow. Comprehensive tests have been added to ensure no regressions.
